integrations/access/common/bot.go

```diff
@@ -29,8 +30,10 @@ import (
 type MessagingBot interface {
 	// CheckHealth checks if the bot can connect to its messaging service
 	CheckHealth(ctx context.Context) error
-	// Broadcast sends an access request message to a list of Recipient
-	Broadcast(ctx context.Context, recipients []Recipient, reqID string, reqData pd.AccessRequestData) (data SentMessages, err error)
+	// AccessListReviewReminder will send a review reminder that an access list needs to be reviewed.
+	AccessListReviewReminder(ctx context.Context, recipients []Recipient, accessList *accesslist.AccessList) error
```

Comment:
```suggestion
	SendReviewReminders(ctx context.Context, recipients []Recipient, accessList *accesslist.AccessList) error
```

The other functions in this interface describe actions (_check_ health, _broadcast_ message, _post_ reply). This function is just a noun. Let's fix that.